src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1812,6 +1816,7 @@ internal bool CanMakeNewHomeWorkspace(object parameter)
 
         private void CloseHomeWorkspace(object parameter)
         {
+            //TODO potentially dispose viewModel here.
```

Comment:
Lately I have been adding "// TODO, QNTM-XXXX:" instead of just "// TODO" so that there is a connection between the todo and why it was created. It also makes it searchable by number. I'm not sure this is really necessary (or a good thing, though it seems good to me, opinions requested).